src/redis-cli.c

```diff
@@ -4708,6 +4709,43 @@ static clusterManagerNode *clusterManagerGetSlotOwner(clusterManagerNode *n,
 static int clusterManagerSetSlot(clusterManagerNode *node1,
                                  clusterManagerNode *node2,
                                  int slot, const char *status, char **err) {
+    /* A new CLUSTER SETSLOT variant is introduced in Redis
+     * 7.2+ to help mitigate the single-point-of-failure
```

Comment:
This comment could mention what this new SETSLOT variant is.

Reply:
Done